app/main.py

```diff
@@ -14,4 +14,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+    for char in phrase:
+        if char.lower() == letter.lower():
+            counter += 1
+    return counter
```

Comment:
Rewrite your code to one line with using function `sum`

Reply:
def count_letter(phrase, letter):
    return sum(1 for char in phrase if char.lower() == letter.lower())